[2021-09-23 22:54:00,397] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-08-29T00:00:00+00:00 [queued]>
[2021-09-23 22:54:00,405] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_2 2021-08-29T00:00:00+00:00 [queued]>
[2021-09-23 22:54:00,405] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-09-23 22:54:00,405] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-09-23 22:54:00,405] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-09-23 22:54:00,432] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): runme_2> on 2021-08-29T00:00:00+00:00
[2021-09-23 22:54:00,434] {standard_task_runner.py:52} INFO - Started process 4455 to run task
[2021-09-23 22:54:00,442] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_2', '2021-08-29T00:00:00+00:00', '--job-id', '118', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/sandbox/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmp7uslkjrn', '--error-file', '/tmp/tmpovej5hlq']
[2021-09-23 22:54:00,442] {standard_task_runner.py:77} INFO - Job 118: Subtask runme_2
[2021-09-23 22:54:00,533] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_2 2021-08-29T00:00:00+00:00 [running]> on host airflowvm
[2021-09-23 22:54:00,650] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_2
AIRFLOW_CTX_EXECUTION_DATE=2021-08-29T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-29T00:00:00+00:00
[2021-09-23 22:54:00,651] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-09-23 22:54:00,652] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'echo "example_bash_operator__runme_2__20210829" && sleep 1']
[2021-09-23 22:54:00,658] {subprocess.py:75} INFO - Output:
[2021-09-23 22:54:00,662] {subprocess.py:79} INFO - example_bash_operator__runme_2__20210829
[2021-09-23 22:54:01,664] {subprocess.py:83} INFO - Command exited with return code 0
[2021-09-23 22:54:01,696] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20210829T000000, start_date=20210923T225400, end_date=20210923T225401
[2021-09-23 22:54:01,749] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-09-23 22:54:01,784] {local_task_job.py:151} INFO - Task exited with return code 0
